data/pigui/modules/station-view/07-police.lua

```diff
@@ -1,17 +1,157 @@
+local InfoFace = import 'ui/PiguiFace'
+local Format = import "Format"
+local Character = import "Character"
 local l = Lang.GetResource("ui-core")
 
+local ModalWindow = import 'pigui/libs/modal-win.lua'
+
+local rescaleVector = ui.rescaleUI(Vector2(1, 1), Vector2(1600, 900), true)
+
+local pionillium = ui.fonts.pionillium
+local orbiteer = ui.fonts.orbiteer
+local colors = ui.theme.colors
+local face = nil
+local stationSeed = false
+
+local popup = ModalWindow.New('policePopup', function()
+	ui.text(l.YOU_NOT_ENOUGH_MONEY)
+	ui.dummy(Vector2((ui.getContentRegion().x - 100*rescaleVector.x) / 2, 0))
+	ui.sameLine()
+	if ui.button(l.OK, Vector2(100*rescaleVector.x, 0)) then
+		self:close()
+	end
+end)
+
+local gray = Color(76, 76, 76)
+
+local widgetSizes = ui.rescaleUI({
+	itemSpacing = Vector2(4, 9),
+	windowPadding = Vector2(14, 14),
+	faceSize = Vector2(586,565),
+}, Vector2(1600, 900))
+
+widgetSizes.itemSpacing = Vector2(math.ceil(widgetSizes.itemSpacing.x),
+								  math.ceil(widgetSizes.itemSpacing.y))
+widgetSizes.windowPadding = Vector2(math.ceil(widgetSizes.windowPadding.x),
+									math.ceil(widgetSizes.windowPadding.y))
+
+
+local function payfine(fine)
+	if Game.player:GetMoney() < fine then
+		popup:open()
+		return
+	end
+
+	Game.player:AddMoney(-fine)
+	Game.player:ClearCrimeFine()
+end
+
+
+local function crime_record()
+	local past_crimes, stump = Game.player:GetCrimeRecord()
+	if #utils.build_array(pairs(past_crimes)) > 0 then
+		ui.withFont(orbiteer.xlarge.name, orbiteer.xlarge.size,
+					function() ui.textColored(gray, l.CRIMINAL_RECORD) end)
+		ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size,
+					function ()
+						for k,v in pairs(past_crimes) do
+							ui.textColored(gray, v.count.."\t"..Legal.CrimeType[k].name)
+						end
+					end)
+	end
+end
+
+
+local function outstanding_fines()
+	local crimes, fine = Game.player:GetCrimeOutstanding()
+	if #utils.build_array(pairs(crimes)) > 0 then
+
+		-- headline
+		ui.withFont(orbiteer.xlarge.name, orbiteer.xlarge.size,
+					function() ui.text(l.OUTSTANDING_FINES) end)
+		-- wrap list in medlarge font
+		ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size,
+					function()
+						for k,v in pairs(crimes) do
+							ui.text(v.count.."\t"..Legal.CrimeType[k].name)
+						end
+						local pay_fine_text = string.interp(l.PAY_FINE_OF_N, { amount = Format.Money(fine) })
+						ui.text(pay_fine_text)
+						ui.sameLine()
+						if ui.button(l.PAY, Vector2(100,0)) then
+							payfine(fine)
+						end
+					end)
+	else
+		ui.withFont(pionillium.large.name, pionillium.large.size,
+					function() ui.text(l.WE_HAVE_NO_BUSINESS_WITH_YOU) end)
+	end
+end
+
+
+local function drawPolice()
+
+	-- Todo: add to language translation system
+	local intro_txt = "This is the ".. Game.system.faction.policeName ..
+		" of the "  .. Game.system.faction.name .. "."
+
+	ui.withFont(pionillium.large.name, pionillium.large.size,
+				function () ui.text(intro_txt) end)
```

Comment:
I'd move this inside the "CrimeStats" child window so window padding is applied

Reply:
Yes, that looks way better, thanks.